src/model/game/GameManager.ts

```diff
@@ -836,23 +26,54 @@ export default class GameManager {
 
     // Used for loading game specific settings before game is selected.
     static get defaultGame(): Game {
-        return this._gameList.find(value => value.internalFolderName === "RiskOfRain2")!;
+        return this.gameList.find(value => value.internalFolderName === "RiskOfRain2")!;
     }
 
     static get gameList(): Game[] {
-        return [...this._gameList];
+        return EcosystemSchema.r2mm_games.map((game) => {
+            const r2mm = game.r2modman!;
+
+            // Map distribution values onto the r2mm StorePlatformMetadata type.
+            // The `identifier` property may not be present, so we check it beforehand to avoid JSON import lint errors.
+            const distributions = game.distributions.map((x: any) => new StorePlatformMetadata(
+                    GetStorePlatformFromName(x.platform),
+                    x.identifier!
+                )
+            );
+
+            const displayMode = (GameSelectionDisplayMode as any)[(r2mm.gameSelectionDisplayMode as string).toUpperCase()];
+            const instanceType = (GameInstanceType as any)[(r2mm.gameInstanceType as string).toUpperCase()];
+            const installerVariant = GetInstallerVariantFromName(r2mm.packageLoader) as PackageLoader;
+
+            return new Game(
+                game.meta.displayName,
+                r2mm.internalFolderName as GAME_NAME,
+                r2mm.settingsIdentifier as string,
+                r2mm.steamFolderName as string,
+                r2mm.exeNames as string[],
+                r2mm.dataFolderName as string,
+                r2mm.packageIndex as string,
+                r2mm.exclusionsUrl as string,
```

Comment:
Note that exclusions are no longer defined in the Game object.

Reply:
Will fix in ecosystem-schema and then propagate those changes over to here.

Reply:
What ever suits you but I guess we could just ignore the field here even if it's included in the data?